From dc80e194e93e6f0e25c8ad3e015d83aca2a99e10 Mon Sep 17 00:00:00 2001
From: Steve Lhomme <slhomme@matroska.org>
Date: Sat, 24 Feb 2024 08:38:00 +0100
Subject: [PATCH] drop CMake older than 3.5

It generates a warning with newer versions of CMake.
3.4 was released in 2015. We require C++17 with compilers only available in 2017/2018.
If your CMake is older than your toolchain, you have a problem.

There's no need to support legacy CMake going forward in libebml 2/libmatroska 2.

Upstream: https://github.com/Matroska-Org/libmatroska/commit/dc80e194e93e6f0e25c8ad3e015d83aca2a99e10

Signed-off-by: Bernd Kuhls <bernd@kuhls.net>
[backported to version 1.7.1]
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 221c8ac6..97b392e0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1.2)
+cmake_minimum_required(VERSION 3.5)
 
 project(matroska VERSION 1.7.1)
 
